docs/pages/installation.mdx

```diff
@@ -367,14 +367,15 @@ the Teleport installation by setting configuration variables in the
 `/etc/teleport.d/conf` file on the EC2 instance. See the [Starter Cluster Configuration Template](https://github.com/gravitational/teleport/tree/branch/v(=teleport.major_version=)/examples/aws/terraform/starter-cluster/data.tpl)
 for a list of the available configuration options.
 
-The image names all include the build timestamp (shown as `$TIMESTAMP` in the table
-below), and are tagged for easier searching.
+The image names all include the architecture (shown as `$ARCH` in the table
+below) and the build timestamp (shown as `$TIMESTAMP`), and are tagged for
+easier searching.
 
 | Image name | Edition | FIPS support | AMI Tags | Owner Account ID |
 | - | - | - | - | - |
-| `teleport-oss-(=teleport.version=)-$TIMESTAMP` | OSS | No | `TeleportVersion: (=teleport.version=)`, `TeleportEdition: oss`, `TeleportFipsEnabled: false` | 146628656107 |
-| `teleport-ent-(=teleport.version=)-$TIMESTAMP` | Enterprise | No | `TeleportVersion: (=teleport.version=)`, `TeleportEdition: ent`, `TeleportFipsEnabled: false` | 146628656107 |
-| `teleport-ent-(=teleport.version=)-fips-$TIMESTAMP` | Enterprise | Yes | `TeleportVersion: (=teleport.version=)`, `TeleportEdition: ent`, `TeleportFipsEnabled: true` | 146628656107 |
+| `teleport-oss-(=teleport.version=)-$ARCH-$TIMESTAMP` | OSS | No | `TeleportVersion: (=teleport.version=)`, `TeleportEdition: oss`, `TeleportFipsEnabled: false` | 146628656107 |
+| `teleport-ent-(=teleport.version=)-$ARCH-$TIMESTAMP` | Enterprise | No | `TeleportVersion: (=teleport.version=)`, `TeleportEdition: ent`, `TeleportFipsEnabled: false` | 146628656107 |
+| `teleport-ent-(=teleport.version=)-$ARCH-fips-$TIMESTAMP` | Enterprise | Yes | `TeleportVersion: (=teleport.version=)`, `TeleportEdition: ent`, `TeleportFipsEnabled: true` | 146628656107 |
```

Comment:
I'd probably expand this to all 6 rows instead of templating some things (arch) but not others (oss/ent and fips).

Should we include the architecture tag in the `AMI Tags` column?

I don't feel super strongly about it if you like the current format.

Reply:
Also -- this looks like a minorly breaking change, as patterns that would have matched the previous fips ami names no longer will.   Since ARCH immediately proceeds TIMESTAMP, the non-fips images will probably cheat in via a wildcard.

Basically, consider what https://github.com/gravitational/teleport/pull/36112/files#diff-5442869deeb8c3e1e670605f82d3ed05fc3b9db2873b483ff7efd636117baeefR103 would look like for consumers of these AMIs before and after.

Reply:
Expanded to 6 rows, didn't include the tag (as it doesn't match the other formats), and updated changelog.